index.html

```diff
@@ -35,7 +35,7 @@
             <div id="tagline">
                <h4>Welcome to Vanilla Squad's blog</h4>
                <p>
-                  We are four intrepid JavaScript Programmers enrolled in Founders And Coders
+                  We are four intrepid JavaScript Programmers enrolled in Founders & Coders
```

Comment:
From [what I've read](http://stackoverflow.com/questions/3493405/do-i-really-need-to-encode-as-amp) it seems to be better to escape ampersands as "&amp;"

I'll make this change in my next PR